src/v/redpanda/application.cc

```diff
@@ -1764,6 +1764,35 @@ void application::start_runtime_services(
                 }
             });
       });
+    ssx::background = feature_table.invoke_on_all(
+      [this](features::feature_table& ft) {
+          return ft.await_feature(features::feature::rpc_transport_unknown_errc)
+            .then([this] {
+                if (ss::this_shard_id() == 0) {
+                    vlog(
+                      _log.info, "All nodes support unknown RPC error codes");
```

Comment:
Since this message will be on every startup on modern systems, maybe debug level is better (it is kind of a hard message for a user to understand at info level)

Reply:
Done